staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -579,6 +579,8 @@ type APIResource struct {
 	// verbs is a list of supported kube verbs (this includes get, list, watch, create,
 	// update, patch, delete, deletecollection, and proxy)
 	Verbs Verbs `json:"verbs" protobuf:"bytes,4,opt,name=verbs"`
+	// shortName is the suggested short name of the resource.
+	ShortName string `json:"shortName,omitempty" protobuf:"bytes,5,opt,name=shortName"`
```

Comment:
Last time this came up I think I argued that this should be a list, otherwise it's hard for people to ever change it.